MapboxNavigationTests/Fixtures.xcassets/us-101.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "filename" : "us-101@3x.png",
```

Comment:
We should consider adding the `us-101.png` (aka 1x.png) and `us-101@2x.png`

Reply:
Yep, this should be part of parallelizing tests for multiple devices. IIRC, it was too flaky with Xcode 8 but we should give it another try.

Reply:
Ticketed out in #1233

Reply:
@vincethecoder the reason why you don't see `foo@1x.png` assets in our project is because those are for the iPhone 2G/3G/3GS platforms, which we don't support.

Reply:
Oh okay. Good to know.